src/language-js/comments.js

```diff
@@ -623,8 +623,11 @@ function handleLastFunctionArgComments({
   return false;
 }
 
-function handleImportSpecifierComments({ comment, enclosingNode }) {
-  if (enclosingNode?.type === "ImportSpecifier") {
+function handleModuleSpecifierComments({ comment, enclosingNode }) {
```

Comment:
Sorry, I thought this was the function added in #12746, can we move this part of logic into `handleModuleSpecifiersComments` function?

Reply:
Sound good. 
I fixed.